web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
> The exception are models like sale.order.line or other one2many models that can go in the same file as sale.order.

@pedrobaeza - That exception doesn't seem to be in our guidelines. I interpreted it as a hard 1:1 rule, which I fully agree with.

@rvalyi Komodo does this as well